internal/server/server.go

```diff
@@ -447,6 +447,12 @@ func (s *Server) startHTTPServer(ctx context.Context, l net.Listener, grpcSrv *g
 
 	s.pool.Go(func(_ context.Context) error {
 		log.Infof("Starting HTTP server at %s", s.conf.HTTPListenAddr)
+
+		if suggestHub {
+			log.Info("Cerbos Hub offers features like enhanced policy management, " +
```

Comment:
I don't think this should be logged by the HTTP logger although I guess it needs to be here in order to be logged after the server has started.

```suggestion
			zap.L().Named("hub").Info("Cerbos Hub offers features like enhanced policy management, " +
```